src/openrct2/management/NewsItem.cpp

```diff
@@ -295,6 +295,20 @@ std::optional<CoordsXYZ> news_item_get_subject_location(int32_t type, int32_t su
     return subjectLoc;
 }
 
+static NewsItem* news_item_first_open_slot()
```

Comment:
```suggestion
static NewsItem* news_item_first_open_queue_slot()
```
There is also a similar function for finding a slot in the history, so this hopefully keeps them separated better.